app/models/recipe.rb

```diff
@@ -1,3 +1,10 @@
 class Recipe < ApplicationRecord
+   has_many :recipe_ingredients
+   has_many :ingredients, through: :recipe_ingredients
+   
    validates_presence_of :name, :complexity, :genre
+
+   def total_cost
+      ingredients.pluck(:cost).sum
```

Comment:
I see what you're going for here - you want to separate the `cost` attribute from each ingredient, then add them all together. However, this is a _Ruby_ way of doing this calculation, when the ActiveRecord method `.sum(:attr)` works much better and at scale. So instead of this, it should be: 
```ruby
ingredients.sum(:cost)
```